<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Peter Hedlund &lt;peter.hedlund@kdemail.net&gt;</author>
 <class>ClearDialogBase</class>
 <widget class="QWidget" name="ClearDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>153</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="clearButtonGroup">
     <property name="title">
      <string>Selection</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="clearAllCheckBox">
        <property name="toolTip">
         <string>Delete all information in selected entries</string>
        </property>
        <property name="whatsThis">
         <string>When checked all information will be deleted from the selected entries</string>
        </property>
        <property name="text">
         <string>Clear &amp;All</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="topVerticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>6</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="clearTextCheckBox">
        <property name="toolTip">
         <string>Delete the text from selected entries</string>
        </property>
        <property name="whatsThis">
         <string>When checked the text will be deleted from selected entries</string>
        </property>
        <property name="text">
         <string>&amp;Text</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="clearImageCheckBox">
        <property name="toolTip">
         <string>Delete the image link from selected entries</string>
        </property>
        <property name="whatsThis">
         <string>When checked the image link will be deleted from selected entries</string>
        </property>
        <property name="text">
         <string>&amp;Image Links</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="clearSoundCheckBox">
        <property name="toolTip">
         <string>Delete the sound link from selected entries</string>
        </property>
        <property name="whatsThis">
         <string>When checked the sound link will be deleted from selected entries</string>
        </property>
        <property name="text">
         <string>&amp;Sound Links</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="bottomVerticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>69</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
